torch/onnx/symbolic_registry.py

```diff
@@ -16,6 +17,17 @@
     module = importlib.import_module('torch.onnx.symbolic_opset{}'.format(opset_version))
     _symbolic_versions[opset_version] = module
 
+def register_quantized_ops(domain, version):
```

Comment:
I would suggest to move this function to symbolic_caffe2, since this will only be used there.